src/commands/scanner/run.ts

```diff
@@ -99,8 +99,22 @@ export default class Run extends ScannerCommand {
 			char: 'v',
 			description: messages.getMessage('flags.vceDescription'),
 			longDescription: messages.getMessage('flags.vceDescriptionLong'),
+			exclusive: ['json'],
+			deprecated: {
+				messageOverride: messages.getMessage('flags.vceParamDeprecationWarning')
+			}
+		}),
+		'severity-for-error': flags.number({
+            char: 's',
+            description: messages.getMessage('flags.sfeDescription'),
+            longDescription: messages.getMessage('flags.sfeDescriptionLong'),
 			exclusive: ['json']
```

Comment:
Should the `severity-for-error` and `violations-cause-error` flags be mutually exclusive?

Reply:
Using -s wouldn't do anything additional if you already used -v, so I can make them mutually exclusive.